components/color_input.jsx

```diff
@@ -91,6 +94,8 @@ class ColorInput extends React.Component {
                     <div
                         ref={this.getColorPicker}
                         className='color-popover'
+                        data-test-id='ChromePickerModal'
```

Comment:
As mentioned over the DM, this will result in multiple IDs in a modal.  I'd suggest to generate dynamic IDs for each `ColorIput` by passing `id` from `ColorChooser` component, then prepending that to each ID.  Something like:
- ``id={`${id}Input`}``
- ``id={`${id}Button`}``
- ``id={`${id}ButtonColor`}``
- ``id={`${id}ChromePicker`}``

Reply:
done 💃